### PR TITLE
fixed null bug on drink ingredients

### PR DESCRIPTION
**Pull request by davidhdaw**
What does this PR do?
-Not duplication protection ironically. It actually avoids the null bug on drink ingredients
Where should your reviewer start?
-line 28 of randomView.vue
What (if any) features are you implementing?
-Just the null check
What (if anything) did you refactor?
-our getNewDrink function
Were there any issues that arose?
-Duplication testing proved EXTREMELY hard to test given the limited api calls we can make in one minute. Decided it was an out of scope edge case.
Is there anything that you need from your teammate?
-Nope

### CODE REVIEW

File: src/views/RandomView.vue [hunk 1]
```diff
@@ -25,8 +25,13 @@ import { ref } from "vue"
       const ingredientsList = [];
       for (let i = 1; i < 16; i++) {
         if (data.drinks[0][`strIngredient${i}`]) {
-          const ingredient = data.drinks[0][`strMeasure${i}`] + " " + data.drinks[0][`strIngredient${i}`]
-          ingredientsList.push(ingredient)
+          if (!data.drinks[0][`strMeasure${i}`]) {
```
Comment (Cyanne-Jones) [inline src/views/RandomView.vue:28]: This is such a rad way to check that it exists and make sure the right measurement goes with the right ingredient.

**Review by Cyanne-Jones — APPROVED** [review]
This looks awesome David, this is such a smart way to squash this bug and I appreciate you knocking it out so quickly!
